src/plugins/data_explorer/public/components/app_container.scss

```diff
@@ -1,8 +1,9 @@
 $osdHeaderOffset: $euiHeaderHeightCompensation;
 
 .deSidebar {
-  max-width: 462px;
-  min-width: 400px;
+  max-width: 620px;
+  min-width: 180px;
```

Comment:
I am not a fan of these. Wouldn't they limit how far I can narrow or expand the panels? Wouldn't having a `min` prevent me from totally collapsing the left-bar if i desired?

Reply:
We any way have buttons for collapse and expand, this will not prevent  from completely closing it

Reply:
onClick of the button , the side bar will be collapsed.

https://github.com/opensearch-project/OpenSearch-Dashboards/assets/116718597/2b18f51f-07a5-43c1-9f02-607aa012b1b4

Reply:
Can we eliminate these two styles. They dont seem necessary anymore